eos-upgrade-eos2-to-eos3

```diff
@@ -106,16 +106,17 @@ if ! $FORCE; then
     fi
 fi
 
-# Stop any updates in progress
-killall ostree-daemon || true
+# Stop any updates in progress, and prevent the timer from restarting them
+systemctl stop eos-updater.timer || true
+systemctl stop eos-updater.service || true
```

Comment:
As @dbnicholson suggested in https://github.com/endlessm/eos-customer-support/pull/5/commits/5a229b823ae2b0338cd0483f44290af976943229#r91096639, I think these should be:
```
systemctl stop eos-autoupdater.timer || true
systemctl stop eos-autoupdater.service || true
```
I checked that in my 2.6.10 VM and that seems to be indeed the name of the units, not `eos-updater.<service|timer>`